book/command-line-parsing/dune

```diff
@@ -1,15 +1,9 @@
-(rule
- (target dune.gen)
- (action
-  (with-stdout-to %{target}
-   (run ocaml-mdx rule --locks ../../global-lock --duniverse-mode --direction=to-md %{dep:README.md}
-    --prelude prelude.ml))))
-
-(alias
- (name   runtest)
- (deps   README.md)
- (action (diff dune.inc dune.gen)))
-
-(include dune.inc)
+(mdx
+ (files README.md)
+ (packages
+  core
+  mdx
```

Comment:
How come the `mdx` stanza depends on `mdx`? :-)

Reply:
That's because of the two binary issue, I'll ask the dune team if there's a better way to work around that until we have a native toplevel!